extensions/GoogleAnalytics/template/en/default/hook/global/header-additional_header.html.tmpl

```diff
@@ -0,0 +1,28 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% RETURN UNLESS Bugzilla.params.google_analytics_tracking_id && !user.in_group(Param('insidergroup')) %]
+
```

Comment:
we shouldn't load GA on provide bugs.  this probably means

`[% RETURN IF bug.defined && bug.groups_in.size %]`